### DISCUSSION TITLE
Model name and Base Auth in credentials

### DISCUSSION THREAD

**Discussion by arberrexhepi** [FlowiseAI/Flowise#1035] [Ideas]
It would be great to be able to store a collection of model names and base auth paths because entering them manually is quite tedious especially since when working with the field if you press Backspace when not on field focus the UI deletes the component, and there is no undo button so it makes it even more complicated.

**Comment by arberrexhepi** [comment]
it's either that, or some sort Project Config

1. you can create a new type of flow (something like a "Config Flow")
- Add service components, set auth paths, credentials (as is already implemented)
- Connect that to sub config components to set different collection names, additional paths so on
- No connections necessary, just a managing canvas 
2. Accessible via Dashboard Tab "Project Config" (ie kind of like the new credentials config tab in dashboard)
- To seperate form regular Chat Flows 
3. Referencing these Project Config Flows
- component added to the sidebar in Canvas
- drop a reference (for ex, drag and drop the Zep config, or Pinecone, etc)
- select the specific config required (i.e. do you want to add the memory node, or upsert, or retrieve, and so on)

**Comment by HenryHengZJ** [comment]
We're thinking of creating a Global Env Variable so you can use it within the canvas nodes
